src/main/webapp/WEB-INF/tags/languageSwitch.tag

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<%@ taglib prefix="spring" uri="http://www.springframework.org/tags" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+<%@ tag description="Language Switcher Tag" pageEncoding="UTF-8" %>
+
+<spring:message code="public.common.language.switch.locale" var="languageSwitchLocale"/>
+<script>
```

Comment:
I believe idam-web-public has to be usable without JS enabled ....

Reply:
That is something I was afraid of. If this is the case, I will rewrite this to happen server-side in java.

Reply:
This requirement comes from GDS (Government Digital Services)
https://www.gov.uk/service-manual/technology/using-progressive-enhancement#start-with-html

Reply:
Fixed.